packages/api/helper/mailer.js

```diff
@@ -36,13 +36,22 @@ const email = new emailTemplate({
   preview: false,
 });
 
+
+const recEmail = (nomEmail) => {
+  if(process.env.RECIPIENT_EMAIL !== null && process.env.RECIPIENT_EMAIL.length > 0){
```

Comment:
Do you have prettier installed and format on save enabled? Probably want to do that, it'll help make the formatting consistent. Here's an article about it if you're not sure what I'm talking about: https://scottsauber.com/2017/06/10/prettier-format-on-save-never-worry-about-formatting-javascript-again/